packages/material-ui/src/ButtonBase/focusVisible.js

```diff
@@ -1,75 +1,111 @@
-import warning from 'warning';
-import ownerDocument from '../utils/ownerDocument';
```

Comment:
Propbably need to pass the owner document in the `prepare` function. Otherwise I'm not sure how this works with iframes.